locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
```

Comment:
Is `locals` being used to initialize `cluster_name` here because `var.environment` can't be used in `variables.tf`?

Reply:
yes, it can not solve interpolations in the `variables.tf`,  

This is not going to work.
```
variable "cluster_name" {
  default = "ecs${title(var.environment)}Cluster"
}
```
I am not sure if there another way to achieve this, let me know if I am missing something.

Reply:
I tested this with an existing Terraform project, and it works well. Also, your defaults help preserve backwards compatibility (which is always nice).